sample/android-sdk-sample/app/src/main/java/com/bitmark/androidsdksample/samples/StoreKeySample.java

```diff
@@ -0,0 +1,66 @@
+package com.bitmark.androidsdksample.samples;
+
+import android.app.Activity;
+import android.content.Context;
+
+import com.bitmark.apiservice.utils.callback.Callback0;
+import com.bitmark.apiservice.utils.callback.Callback1;
+import com.bitmark.sdk.authentication.KeyAuthenticationSpec;
+import com.bitmark.sdk.features.Account;
+
+import io.reactivex.Single;
+
+import static com.bitmark.androidsdksample.utils.KeyUtil.getEncryptionKeyAlias;
+import static com.bitmark.androidsdksample.utils.KeyUtil.getRandomEncryptionKeyAlias;
+import static com.bitmark.androidsdksample.utils.KeyUtil.saveAccountNumber;
+import static com.bitmark.androidsdksample.utils.KeyUtil.saveEncryptionKeyAlias;
+
+public class StoreKeySample {
+    public static final int KEY_VALID_DURATION = 600; // 10 * 60s
+
+    public static void storeAccountToKeystore(Account account, Context context, Activity activity) {
+        String accountNumber = account.getAccountNumber();
+        final String encryptionKeyAlias = getRandomEncryptionKeyAlias(accountNumber);
+
+        KeyAuthenticationSpec spec = new KeyAuthenticationSpec.Builder(context)
+                .setKeyAlias(encryptionKeyAlias)
+                .setAuthenticationRequired(false)
```

Comment:
`.setAuthenticationRequired(false)` makes the `.setAuthenticationValidityDuration(KEY_VALID_DURATION)` is unusable.